app/src/main/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenter.java

```diff
@@ -2,17 +2,27 @@
 
 import android.support.annotation.NonNull;
 
+import java.util.Arrays;
+import java.util.List;
+
 import io.intrepid.contest.R;
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Category;
 import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.models.ScoreWeight;
 import io.reactivex.disposables.Disposable;
 import timber.log.Timber;
 
 class ContestOverviewPresenter extends BasePresenter<ContestOverviewContract.View>
         implements ContestOverviewContract.Presenter {
 
     private static final int TEMPORARY_NUM_SUBMISSIONS_WAITING = 3;
+    private static final List<ScoreWeight> scoreWeights = Arrays.asList(new ScoreWeight(1, R.string.poor),
```

Comment:
Double check that you ran the formatter on this page. Not sure how we have multiple lines setup and I just want to make sure this is configured properly.